DTK_MeshContainer.hpp: No such file or directory

Dear all, 

Sorry for bothering. Could you please help me install libmesh?

I am using Windows 10 64 bit, Windows Subsystem Linux, Ubuntu 18.04. I have done:

```
git clone git://github.com/libMesh/libmesh.git 
cd libmesh
./configure
make
```

The process went on perfect until this happened:

```
In file included from ./include/libmesh/dtk_adapter.h:27:0,
                 from src/solution_transfer/dtk_adapter.C:24:
./include/libmesh/dtk_evaluator.h:31:10: fatal error: DTK_MeshContainer.hpp: No such file or directory
 #include <DTK_MeshContainer.hpp>
          ^~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
Makefile:16028: recipe for target 'src/solution_transfer/libmesh_dbg_la-dtk_adapter.lo' failed
make[1]: *** [src/solution_transfer/libmesh_dbg_la-dtk_adapter.lo] Error 1
make[1]: Leaving directory '/mnt/x/libmesh/libmesh'
Makefile:29399: recipe for target 'all-recursive' failed
make: *** [all-recursive] Error 1
```

And I checked that `DTK_MeshContainer.hpp` is not in the source code.

Was I doing something wrong? Any hint would be greatly appreciated.

Best,
Shawn

--- a/configure
+++ b/configure
@@ -36531,7 +36531,7 @@ else
   enabledtk=no
 fi
 
-          if test "$enabledtk" != "xno"; then :
+          if test "x$enabledtk" != "xno"; then :
 
                   enabledtk=yes
 
@@ -36551,6 +36551,7 @@ fi
           enableifpack=no
 
           enableepetraext=no
+  enableepetra=no
 
 
 

--- a/m4/trilinos.m4
+++ b/m4/trilinos.m4
@@ -365,7 +365,7 @@ AC_DEFUN([CONFIGURE_TRILINOS_9],
                 [DTK_MAKEFILE_EXPORT=$withdtkdir/packages/dtk/Makefile.export.Makefile.export.DataTransferKit],
                 [enabledtk=no])
 
-          AS_IF([test "$enabledtk" != "xno"],
+          AS_IF([test "x$enabledtk" != "xno"],
                 [
                   enabledtk=yes
                   AC_DEFINE(TRILINOS_HAVE_DTK, 1, [Flag indicating whether the library shall be compiled to use the DataTransferKit])
@@ -387,6 +387,7 @@ AC_DEFUN([CONFIGURE_TRILINOS_9],
   dnl anyone ever wants to go back and write a configure test with
   dnl an older Trilinos, that would be great!
   enableepetraext=no
+  enableepetra=no
 
 
   AC_SUBST(AZTECOO_MAKEFILE_EXPORT)

